system_notification/infra/jwt/jose_jwt_adapter.py

```
@@ -0,0 +1,30 @@
+    def encode(self, data: Dict[JWT_TYPES, JWT_TYPES]) -> str:
+        return self._jwt.encode(data, self._secret, algorithm="HS256")
+
+    def decode(
+        self, data: str, verify_signature: bool = True
+    ) -> Dict[JWT_TYPES, JWT_TYPES]:
+        return self._jwt.decode(
+            data, self._secret, options={"verify_signature": verify_signature}
+        )
+
+    def is_valid(self, data: str, verify_signature: bool = True) -> bool:
+        with contextlib.suppress(Exception) as e:
```

Ruff (0.13.3)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)